package.nls.json

```diff
@@ -99,6 +99,7 @@
     "AWS.telemetry.notificationMessage": "The AWS Toolkit collects usage metrics by default. These metrics help drive toolkit improvements. This setting can be changed from the IDE settings.",
     "AWS.telemetry.notificationOk": "OK",
     "AWS.telemetry.notificationViewSettings": "View Settings",
+    "AWS.command.apig.invokeRemoteRestApi": "Invoke on AWS",
```

Comment:
Nit: `Test on AWS`?

it's called "Test Invoke", but `Invoke` reminds me too much of Lambda

Reply:
"Invoke" is intentionally similar to the action for Lambda, so that users see familiar names for similar actions.